Mergin Maps EE docs (#454)
MerginMaps/docs: src/server/administer/index.md

# Administer
 
Administration guide will help you to configure and maintain your <CommunityPlatformNameLink /> or <EnterprisePlatformNameLink />. The main SaaS <DashboardLink desc="Mergin Maps Server"/> is maintained for you by <MainPlatformName /> team. Read more about server platforms in the [overview article](../index.md).
 
[[toc]]

## Server environment

> See list of all [environment variables](./environment.md) that you can configure can be found

Reviewer: ```suggestion
See the list of all [environment variables](./environment.md) that you can configure.
```
